loadtest/config.json

```diff
@@ -5,7 +5,7 @@
     "tls": false,
     "msgs_per_tx": 10,
```

Comment:
Note, we may want to change this default to like 1, since otherwise it will send a TON of messages, whereas before the config fields were named weirdly such that txs per block = 500 and msgs per tx = 10 would result in 50 txs with 10 messages each IIRC. now, it would ACTUALLY result in 500TPS of 10 messages PER tx

Reply:
Did we change the concept here? I think previously if target is 500, batch size is 10, then it will only send 50txs  and each tx will contain 10 messages

Reply:
logic has not changed. will set this to 1 as default